### ISSUE TITLE
Cannot specify base url for GDExtension on web platform

### ISSUE THREAD

**Issue by mmarklar** [godotengine/godot#98272]
### Tested versions

Tested in 4.3-stable

### System information

Web/WASM

### Issue description

I am trying to serve a Godot project on the web and am unable to load GDExtension wasm from urls not relative to the current page. The urls used by the server look something like:
```
https://servername.com/play/
https://servername.com/static/
```
with `play/` running through an application server to template the html file while `static/` files (js, wasm, etc) are served from something like S3. 

This works fine until GDExtensions are involved because you can set `mainPack` and `executable` to the path you want and they will be fetched correctly. For extensions, it will fetch them as a relative path and there does not seem to be anything to change it. Using the urls above it would go to `https://servername.com/play/libname.whatever.wasm` instead of `https://servername.com/static/libname.whatever.wasm`. When I modify the application server to return the wasm on the url Godot uses, everything works.

Three questions:
* Am I missing something? Is there some way to set this?
* Is this a common problem that should be fixed for all godot users or am I on my own here?
* If it is a common problem then any pointers on what the 'correct' way to approach it would be appreciated.

Thanks!

### Steps to reproduce

N/A

### Minimal reproduction project (MRP)

N/A

**Comment by wxfred** [comment]
Yes, the **https://servername.com/play/xxx.js** will try to access 
**https://servername.com/xxx.wasm** (wrong) not 
**https://servername.com/play/xxx.wasm** (correct)
please give us an option to change the base url.

**Comment by wxfred** [comment]
I found a solution, change the **executable** to add your xxx.js path like
`const GODOT_CONFIG = {"args":[],"canvasResizePolicy":2,"executable":"play/xxx",...`
will make the engine load all files correctly.
But i don't know where to change **executable** in the godot editor.
